include/IECore/MeshAlgo.h

```diff
@@ -54,6 +54,8 @@ std::pair<PrimitiveVariable, PrimitiveVariable> calculateTangents( const MeshPri
 	const std::string &position = "P"
 );
 
+PrimitiveVariable resamplePrimitiveVariable( const MeshPrimitive *mesh, const std::string &primitiveVariable, PrimitiveVariable::Interpolation interpolation );
```

Comment:
Something about this made me expect the primitive variable to be modified in place, and it was only when I read the code that I realised it wasn't. The `const MeshPrimitive` is definitely one clue, but I wonder if calling it resample**d**PrimitiveVariable would be another? I also wonder if passing `const PrimitiveVariable &primitiveVariable` might be more foolproof than passing a name, by forcing the caller to consider the possibility of the non-existence of the variable?